README.md

```diff
@@ -21,6 +21,20 @@ vagrant up
 ./cluster-up.sh
 ```
 
+### Configure kubectl for Kubernetes cluster
+
+```sh
+cp ~/.kube/config ~/.kube/config.bak
+cp config ~/.kube/
+```
+
+### Wait for all nodes to be ready
+
+```sh
+watch -n 1 kubectl get nodes
```

Comment:
I don't have `watch` in my shell.

Reply:
You can remove watch. I thought it was standard in all distros